infra/tf/main.tf

```diff
@@ -229,6 +203,10 @@ data "google_compute_global_address" "api" {
   name = "api-${replace(var.subdomain, ".", "-")}"
 }
 
+data "google_compute_global_address" "bootstrap" {
+  name = "boostrap-${replace(var.subdomain, ".", "-")}"
```

Comment:
Did you want to call this bootstrap?

Reply:
As per above, I kept the current naming to do a test on the containered version. Bootstrap is just how has been named this node currently

Reply:
@rrw-zilliqa What made you think that bootstrap was the incorrect name? I'm not against renaming, but in my mind bootstrap was an accurate name. So I'm worried that we have a different understanding of what this node does.

I thought bootstrap was a good name to describe the node which new nodes contact in order to learn about the existence of the rest of the network.

Reply:
bootstrap is fine. "boostrap" seems odd :-p